File: RiderPointDotnet/RP_Web_API/BLL/PillionService.cs
namespace BLL;
using model;
using dbmanagerUtil;
using System.Collections.Generic;

public static class PillionService
{
    
    public static List<rp_details> GetAllpillion(){
        List<rp_details> lst=new List<rp_details>();
        lst=PillionBDmanager.GetAllpillion();
        return lst;
    }
//     public static bool addStatus(Status_pillion Pillion){
//          return PillionBDmanager.addstatus(Pillion);
//     } 
//     public static bool addPillionRoute(Pillion_route_details Pillion){
//          return PillionBDmanager.addPillionRoute(Pillion);
//     }
    
//     public static List<Pillion_route_details> getbyroute(string start,string end){
//          return PillionBDmanager.getbyroute(start,end);
//     }
//      public static rp_details getPillionbyid(int r){
//          return PillionBDmanager.getPillionbyid(r);
//     }
    
//     public static Status_pillion getStatusbyid(int r){
//          return PillionBDmanager.getStatusbyid(r);
//     }
//       public static Pillion_route_details getRoutebyid(int r){
//          return PillionBDmanager.getRoutebyid(r);
//     } 
    
//     public static bool updatePillion(rp_details r){
//          return PillionBDmanager.updatePillion(r);
//     }
//      //
//     public static bool editStatus(Status_pillion r){
//          return PillionBDmanager.editStatus(r);
//     }
//     public static bool editPillionRoute(Pillion_route_details r){
//          return PillionBDmanager.editPillionRoute(r);
//     }
}

File: RiderPointSpringBoot/Riderpoint/src/main/java/com/demo/Service/PillionService.java
package com.demo.Service;

import java.util.List;

import com.demo.Dto.Pillion_Status_Dto;
import com.demo.Dto.Pillion_route_details_Dto;
import com.demo.Dto.RpDetailDTO;
import com.demo.Model.Pillion_Status;
import com.demo.Model.Pillion_route_details;
import com.demo.Model.RpDetails;

public interface PillionService {
	

	List<RpDetails> getAll();

	Pillion_Status addStatus(Pillion_Status pstatus);

	Pillion_route_details addPillionRoute(Pillion_route_details proute);

	List<Pillion_route_details> getByRoute(String start_point, String end_point);

	RpDetailDTO getPillionById(String id);

	Pillion_Status_Dto getStatusById(String status_id);

	Pillion_route_details_Dto getRouteById(String route_id);

	RpDetails updatePillion(RpDetails pillion);

}
